routes/admin/portals/project-allotment-prof-create/index.js

```diff
@@ -2,12 +2,11 @@ var express = require('express');
 var router = express.Router();
 var fq = require('fuzzquire');
 var projectsModel = fq('schemas/projects');
-var projectHeadModel = fq('schemas/project-heads');
 var adminModel = fq('schemas/admins');
 
 router.get('/', function (req, res, next) {
     try {
-        let usertype = getProfessorType(req.user.email);
+        let usertype = getProfessorType(req.user.email);//usertype returns a promise which contains {admin:true} or {head:true and department: } or {head:false and department: }`
         usertype.then(val => {
             console.log(val);
             if (val.admin) {
```

Comment:
Inside this `if` block, remove the `getDepartment` promise as it is not required. If the user is a `superUser`, display all the projects (Offered by every user). Otherwise, display only the projects offered by that particular user.